webapp/channels/src/components/widgets/modals/components/modal_header.scss

```diff
@@ -0,0 +1,84 @@
+.mm-modal-header {
```

Comment:
I assume this `mm-` prefix is to avoid conflicts with Bootstrap, right? Nothing to change here, but I've been looking more at ripping out Bootstrap lately, and this is making me want to do it even more 😛

Reply:
Yeam mm is a prefix to avoid any conflicts with existing CSS or bootstrap